mne/io/edf/tests/test_edf.py

```diff
@@ -556,6 +567,7 @@ def test_read_latin1_annotations(tmp_path):
 
     with pytest.raises(Exception, match="Encountered invalid byte in"):
         _read_annotations_edf(tal_channel)  # default encoding="utf8" fails
+        _read_annotations_edf(str(annot_file))  # default encoding="utf8" fails
```

Comment:
This line won't be hit because of how `pytest.raises` works (the first statement raises and the `with` execution block stops). It needs its own context manager:
```suggestion
    with pytest.raises(Exception, match="Encountered invalid byte in"):
        _read_annotations_edf(str(annot_file))  # default encoding="utf8" fails
```